docs/source/examples/predict_turbofan_degredation.ipynb

```diff
@@ -250,7 +249,7 @@
     "\n",
     "### Representing the Data\n",
     "\n",
-    "We will represent the data using an entity set. We currently have a single table of records where one engine can many records. This one-to-many relationship can be represented in an entity set by normalizing an entity for the engines. The same can be done for engine cycles. Let's start by structuring the entity set."
+    "Let's start by representing the data with an entity set. This way, we can generate features based on the relational structure of the dataset. We currently have a single table of records where one engine can many records. This one-to-many relationship can be represented by normalizing an engine entity. The same can be done for other one-to-many relationships. We want to make predictions based on the engine, so we will use this engine entity as the target entity for generating features."
```

Comment:
`We currently have a single table of orders where one engine can many orders.`

This sentence is missing a word - maybe this instead `...where one engine can have many orders.`